internal/scripts/installation/compile_MET_all.sh

```diff
@@ -276,6 +281,7 @@ echo "COMPILER = $COMPILER"
 echo "COMPILER_FAMILY = $COMPILER_FAMILY"
 echo "COMPILER_VERSION = $COMPILER_VERSION"
 COMPILER_MAJOR_VERSION=`echo $COMPILER_VERSION | cut -d'.' -f1`
+COMPILER_MINOR_VERSION=`echo $COMPILER_VERSION | cut -d'.' -f2`
```

Comment:
Why was this added? I don't see it used anywhere in the script.

Reply:
Thanks @georgemccabe.  I'll take a closer look at the other comments tomorrow, but I added the COMPILER_MAJOR_VERSION and COMPILER_MINOR_VERSION because it was originally thought that the problem lied with gcc 12.3.0.  It didn't, however, I could see potentially using these in the future so I left them in.